shell/assets/styles/base/_spacing.scss

```diff
@@ -0,0 +1,29 @@
+// Text indent, margins, and paddings from 4-40px in 4px increments
```

Comment:
We have new spacing requirements. I added in these helper classes which closely reflect what we already have but with new values.